backend/build.gradle

```diff
@@ -4,6 +4,7 @@ plugins {
     id 'io.freefair.lombok'
     id 'checkstyle'
     id 'pmd'
```

Comment:
Does this really need to be part of build config? It will slow down builds I assume

Reply:
I think this is the only way to do it, but im not a gradle expert.